hs_event_s3/hsevent/main.py

```diff
@@ -0,0 +1,69 @@
+import redpanda_connect
+import asyncio
+import json
+import django
+import threading
+
+django.setup()
```

Comment:
@devincowan - here is where we pipe S3 events to django to add/remove files. django.setup() must be called before any django module imports. This file is run with rpk to setup a kafka consumer to receive S3 events and update django appropriately. The kafka consumer runs using rpk connect and is spun up as a container.